company/values.md

```diff
@@ -13,3 +13,16 @@ The collection of moments, experiences, and memories that the team shares as we
 ## Progress
 
 The continuous march toward achieving our mission as a team: the milestones, the successes, and the breakthroughs.
+
+## Ethics & Justice First
+
+- We strive to work against and avoid reproducing systemic injustices. 
+- We're grateful and not defensive when people point out our failings here.
+- This is not an attempt to define our exact ethical framework, but documents how thinking about ethics is part of our process and reinforces a culture of talking about ethics rather than avoiding them.
```

Comment:
I still think we can and should enumerate points of ethics that we all agree on. General points like anti-racism/homophobia/transphobia/ableism. Are there business or pricing strategies which we can commit to never using? etc